### PR TITLE
add more languages for india

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

File: cms/models.py [hunk 1]
```diff
@@ -38,8 +40,10 @@
     (THA_TH, 'Thai (Thailand)'),
     (IND_ID, 'Bahasa (Indonesia)'),
     (SPA_ES, 'Spanish (Spain)'),
-    (FRA_FR, 'French (France)'),
+    (FRE_FR, 'French (France)'),
     (POR_PT, 'Portuguese (Portugal)'),
+    (HIN_IN, 'Hindi (India)'),
+    (mar_IN, 'Marathi (India)'),
```
Comment (smn) [inline cms/models.py]: Why lowercase?

**Comment by miltontony** [comment]
@smn ready for review

File: cms/models.py (hunk 1 above)
Comment (miltontony) [inline cms/models.py]: sublime autocompletion..

**Comment by smn** [comment]
:+1: when travis

**Pull request merged**